libs/ui/src/lib/notice/notice.component.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  AfterViewInit,
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+} from '@angular/core';
+import { CviIconName } from '../../../../../dist/libs/icons';
+import { NoticeSeverity } from './notice';
+import { NoticeSeverityToHeaderIconPipe } from './notice-severity-to-header-icon.pipe';
+
+@Component({
+  selector: 'cvi-ng-notice',
+  templateUrl: './notice.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NoticeComponent implements AfterViewInit {
```

Comment:
One idea, looking at the design and data that always comes with the component... Should we rename it to something like `TimedNotice`? Because date and time seem to be always there, and the component is clearly designed to be inside of a some sort of a timeline.

Reply:
Agreed - TimedNotice makes more sense. Renamed